modules/data.remote/R/MODIS_LAI_prep.R

```diff
@@ -0,0 +1,130 @@
+#' Prepare MODIS LAI data for the SDA workflow.
+#'
+#' @param site_info Bety list of site info including site_id, lon, and lat.
+#' @param start_date Start date of SDA workflow.
+#' @param end_date End date of SDA workflow.
+#' @param time_points A vector contains each time point within the start and end date.
+#' @param NCore Number of CPU to be used for LAI extraction.
+#' @param run_parallel Bool variable decide if you want to proceed parallelly.
+#' @param outdir Where the final CSV file will be stored.
+#' @param search_window search window for locate available LAI values.
+#' @param export_csv Decide if we want to export the CSV file.
+#'
+#' @return A data frame containing LAI and sd for each site and each time step.
+#' @export
+#' 
+#' @examples
+#' @author Dongchen Zhang
```

Comment:
If your functions borrow heavily from previous code that you're reorganizing/refactoring, you should acknowledge the previous developers as coauthors

Reply:
I think nearly 90% of this function is written by myself (except for the call_MODIS function calls and the 0.66 threshold)